plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -333,6 +334,24 @@ public void testNumericAggregationPushdown()
         }
     }
 
+    @Test
+    public void testCountDistinctWithStringTypes()
+    {
+        if (!(hasBehavior(SUPPORTS_CREATE_TABLE) && hasBehavior(SUPPORTS_INSERT))) {
+            throw new SkipException("Unable to CREATE TABLE to test count distinct");
```

Comment:
All jdbc connectors do support that?
if so, you can skip the check.

Reply:
Not true for druid